certified-connectors/The Bot Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
@tbpsami thank you so much for fixing the connector files. 
May I ask you please to fix the indentation in the swagger file. all files should be properly formatted. please use the same format you used in the apiProperties.json file.
Please let me know if you have any questions.